python-analysis-server/main.py

```
@@ -195,5 +204,52 @@ async def mark_script(req: MarkRequest):
+        response = model.generate_content(prompt)
+        json_str = response.text.replace("```json", "").replace("```", "").strip()
+        result = json.loads(json_str)
+        generated_script = result.get("generatedScript")
+
+        if not generated_script:
+            raise HTTPException(status_code=502, detail="AI response does not contain generatedScript")
+
+        return {
+            "generatedScript": generated_script
+        }
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)